src/LibVLCSharp.Forms/Shared/IOrientationHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace LibVLCSharp.Forms.Shared
+{
+    /// <summary>
+    /// Force Device Orientation.
+    /// </summary>
+    public interface IOrientationHandler
```

Comment:
Do we really need to force orientation here? To me this is not necessary to implement the lock feature, unless I missed something? :-)

Reply:
Ok, I guess the lock feature isn't only about disabling controls but also about locking orientation!